XamlControlsGallery/DataModel/ControlInfoData.json

```diff
@@ -1038,6 +1038,30 @@
             "Image"
           ]
         },
+        {
+          "UniqueId": "InfoBadge",
+          "Title": "InfoBadge",
+          "Subtitle": "An non-intrusive UI to display notifications or bring focus to an area",
+          "ImagePath": "ms-appx:///Assets/MenuBar.png",
```

Comment:
Is this correct?

Reply:
InfoBadge doesn't really have an asset, so it's just a similar (?) looking one for now. Perhaps this can be updated after we figure out what the icon should be?

Reply:
Icons for new controls is an issue we definitely need to tackle, there already was discussion on this and I think it was leaning towards revamping all icons.